examples/cors.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(async_await)]
+
+use http::header::HeaderValue;
+use tide::middleware::CorsMiddleware;
+
+fn main() {
+    let mut app = tide::App::new();
+
+    app.middleware(
+        CorsMiddleware::new()
+            .allow_origin(HeaderValue::from_static("*"))
+            .allow_methods(HeaderValue::from_static("GET, POST, OPTION"))
+            .echo_back_origin(true),
+    );
+
+    app.at("/").get(async move |_| "Hello, world!");
+
+    app.run("127.0.0.1:8000").unwrap();
+}
```

Comment:
Instead of a new example in the examples dir, this might be nice to include as a doc comment / README example instead!

We have quite a few examples already, and not that many docs. I think it might be good to start changing that balance (:

Reply:
@yoshuawuyts I fixed it!